scripts/cocod

```diff
@@ -7,7 +7,7 @@ import sys
 from coco import Master
 
 def start(args):
-    master = Master(args.conf)
+    master = Master(**vars(args))
```

Comment:
where do these `**vars(args)` come from?

Reply:
I will add a comment about this black magic I copy-pasted from somewhere to get the members of the namespace `args`.